maxim/logger/llamaindex/client.py

```
@@ -0,0 +1,422 @@
+from maxim.logger.components.generation import GenerationToolCall, GenerationToolCallFunction
+
+from .. import Logger
+from ...scribe import scribe
+from ..components import Trace, Span, Generation, GenerationConfigDict 
+from .utils import LlamaIndexUtils
+
+_INSTRUMENTED = False
+_global_maxim_trace: contextvars.ContextVar[Union[Trace, None]] = (
+    contextvars.ContextVar("maxim_trace_context_var", default=None)
+)
+_agent_spans: contextvars.ContextVar[Dict[str, Span]] = (
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)